msdk-io/msdk-io-mzml2/src/main/java/io/github/msdk/io/mzml/MzMLFileExportMethod.java

```diff
@@ -238,15 +238,20 @@ public Void execute() throws MSDKException {
         }
 
         // total ion current CV param
-        if (!(rawDataFile instanceof MzMLRawDataFile) || (rawDataFile instanceof MzMLRawDataFile
-            && !spectrum.getCVValue(MzMLCV.cvTIC).isPresent())) {
-          if (scan.getTIC() != null) {
-            Float tic = scan.getTIC();
-            writeCVParam(xmlStreamWriter,
-                new MzMLCVParam(MzMLCV.cvTIC, String.valueOf(tic), "total ion current", null));
-          }
+        if (scan.getTIC() != null) {
+          Float tic = scan.getTIC();
+          writeCVParam(xmlStreamWriter,
+              new MzMLCVParam(MzMLCV.cvTIC, String.valueOf(tic), "total ion current", null));
         }
 
+        // remove tic value read from the file
```

Comment:
If you want to remove the TIC CV term from the existing MzMLRawDataFile file, it makes more sense to remove it during importing the file, not during exporting a new file (it makes the source file inconsistent, because its CVParam list would change after exporting)